front_end/panels/ai_chat/LLM/LLMProviderRegistry.ts

```
@@ -103,4 +111,233 @@ export class LLMProviderRegistry {
+      return registered;
+    }
+
+    // Fall back to creating temporary instance
+    return this.createTemporaryProvider(providerType);
+  }
+
+  /**
+   * Get storage keys for a provider
+   * Returns the localStorage keys used by the provider for credentials
+   */
+  static getProviderStorageKeys(providerType: LLMProvider): {apiKey?: string; endpoint?: string; [key: string]: string | undefined} {
```

ESLint

[error] Expected a comma.

(@stylistic/member-delimiter-style)